src/webviews/mixins/reporter.ts

```diff
@@ -0,0 +1,59 @@
+/*!
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as Vue from 'vue'
+import { WebviewClientFactory, GlobalProtocol } from '../client'
+
+/**
+ * Mixin for instrumenting components for testing.
+ */
+const reporter: Vue.ComponentOptionsMixin = {
+    created() {
+        if (this.$data === undefined) {
+            return
+        }
+
+        const id = this.$options.id ?? this.id ?? this.$options._unid
+        const name = this.$options.name ?? this.name
+        const isValidTarget = (target?: string) => !target || target === name || target === id
+        const format = (message: string) => `${name ?? 'unknown'} (${id ?? 'unknown'}): ${message}`
+        const client = WebviewClientFactory.create<GlobalProtocol>()
+
+        client.$inspect(event => {
+            if (!isValidTarget(event.target)) {
+                console.log(format(`Ignoring inspection request for ${event.target}`))
```

Comment:
As it stands right now, all `console.log/warn` statements here will get displayed either in a terminal (running tests) or the VS Code Electron developer tools console (essentially invisible). Is it worth messaging these to the backend for `getLogger()` display?